<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(100, 125, 134);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignmentFlag::AlignTop">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 184, 221);</string>
      </property>
      <property name="text">
       <string>CURRENCY CONVERTER</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>393</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>500</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>163</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 184, 221);</string>
       </property>
       <property name="text">
        <string>Local Currency:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>247</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 184, 221);</string>
       </property>
       <property name="text">
        <string>International Currency:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Local_lineEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>70</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color: rgb(252, 240, 255);
	color: rgb(24, 15, 35);
	border: 3px solid #acffc4;
	border-radius: 5px;

}</string>
       </property>
       <property name="placeholderText">
        <string>Enter local amount (Tsh)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="International_lineEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>150</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color: rgb(252, 240, 255);
	color: rgb(24, 15, 35);
	border: 3px solid #acffc4;
	border-radius: 5px;

}</string>
       </property>
       <property name="placeholderText">
        <string>Enter inter... amount</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Convert_pushButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>260</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 3px solid #acffc4;
	border-radius: 5px;
	color: rgb(191, 169, 82);
}

QPushButton:hover{
	background-color: rgb(209, 152, 255);
	border: 3px solid #dacbff;
	border-radius: 5px;
	color: rgb(0, 170, 127);
}</string>
       </property>
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
